gradle/configure-crash-logger.gradle

```diff
@@ -27,24 +27,24 @@ kotlin {
         macosArm64(),
         iosX64(),
         iosArm64(),
-        iosArm32(),
+//        iosArm32(),
```

Comment:
I don't think we should be disabling 32-bit targets

Reply:
That was me. Rick doesn't publish them. We'll have to resolve that at some point soon, but didn't want that to be a blocker.